src/main/java/org/variantsync/diffdetective/diff/difftree/transform/NaiveMovedArtifactDetection.java

```diff
@@ -34,8 +37,8 @@ public void transform(final DiffTree diffTree) {
                 removed = twin.first();
             }
 
-            final DiffNode afterParent = added.getAfterParent();
-            final DiffNode beforeParent = removed.getBeforeParent();
+            final DiffNode afterParent = added.getParent(AFTER);
+            final DiffNode beforeParent = removed.getParent(BEFORE);
```

Comment:
Well, I'm in favor of such changes as long as we know all call sites because it keeps the code cleaner and simpler. Without this change we would have almost three times as much methods in `DiffNode` for literally no advantage on the call sites.
If you're concerned about the time consumption: Using Neovim I developed a workflow for such things which makes this pretty easy and fast.

Reply:
I agree.